src/render_tasks/d3d12_rt_reflection_task.hpp

```diff
@@ -271,8 +275,8 @@ namespace wr
 					data.base_data.out_hitgroup_shader_table[frame_idx],
 					data.base_data.out_miss_shader_table[frame_idx],
 					data.base_data.out_raygen_shader_table[frame_idx],
-					static_cast<std::uint32_t>(std::ceil(scalar * d3d12::GetRenderTargetWidth(render_target))),
-					static_cast<std::uint32_t>(std::ceil(scalar * d3d12::GetRenderTargetHeight(render_target))),
+					static_cast<std::uint32_t>(std::ceil(d3d12::GetRenderTargetWidth(render_target))),
```

Comment:
Why no more scalar?

Reply:
This would resolve the reflections not scaling with the resolution yes

Reply:
Because render_target width and height are already scaled, so you'd always be rendering to a quarter of the render target.